src/dataProtector/protectDataObservable.ts

```diff
@@ -135,16 +135,22 @@ export const protectDataObservable = ({
           name: vName,
```

Comment:
const ownerAddress = await signer.getAddress();

we can catch error : 

const ownerAddress = await signer.getAddress().catch((e: Error) => {
          throw new WorkflowError('Failed to retrieve the owner's address', e);
        });

Reply:
This is not expected to happen in the nominal workflow.
If this happens there is nothing the end user can do, so we will just throw an "unexpected error"